src/Stormpath/DataStore/DefaultDataStore.php

```diff
@@ -334,4 +339,13 @@ private function getQueryString(array $options) {
 
         return $query;
     }
+
+    public function getCacheManager() {
+        return $this->cacheManager;
+    }
+
+    public function getApiKey()
+    {
+        return $this->requestExecutor->getApiKey();
+    }
```

Comment:
It will not work as a protected method because Application does not extend DefaultDataStore at any point.  We get to the data store via the getDataStore() method which only allows us to have public methods.  We can look into other ways of doing this but unless we make apiKey part of a class that Application and all others extend, it will have to be public.